### PR TITLE
[FIX] sale_order_line_description: fix my email address

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

File: sale_order_line_description/README.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ Credits
 Contributors
 ------------
 
-* Alex Comba <email.address@example.org>
+* Alex Comba <alex.comba@example.org>
```
Comment (pedrobaeza) [inline sale_order_line_description/README.rst]: example.org?

Comment (tafaRU) [reply sale_order_line_description/README.rst]: I need a coffee :blush:

**Comment by tafaRU** [comment]
@pedrobaeza, now it should be ok :wink:
Thanks!

**Review by pedrobaeza — APPROVED** [review]
Fast-tracking it

**Pull request merged**